evcxr_jupyter/src/core.rs

```diff
@@ -162,6 +166,14 @@ impl Server {
             *self.latest_execution_request.lock().unwrap() = Some(message.clone());
             let src = message.code();
             execution_count += 1;
+            self.command_history.push({
+                let name = format!("command_{}", execution_count);
+                let src = src.to_string();
+                (
+                    Box::leak(name.into_boxed_str()),
```

Comment:
Is it necessary to leak here, or could we store a `String` instead of a `&'static str`?